public/scripts/main.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(() => {
+  function onKonamiCode(cb) {
+    let input = '';
+    const key = '38384040373937396665';
+
+    document.addEventListener('keydown', (e) => {
+      input += (`${e.keyCode}`);
+
+      if (input === key) {
+        return cb();
+      }
+
+      if (!key.indexOf(input)) return 1;
+
+      input = (`${e.keyCode}`);
+
+      return 1;
+    });
+  }
+
+  onKonamiCode(() => {
+    const img = new Image();
+
+    img.src = `../images/konami/k${Math.floor(Math.random() * 5)}.webp`;
+    img.style.width = '350px';
+    img.style.height = '300px';
+    img.style.transition = '6s all linear';
+    img.style.position = 'fixed';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        img.style.position = "fixed";
```